src/controllers/cartsController.js

```diff
@@ -1,13 +1,14 @@
-const {
-  addItemToCartsService,
-  getCartsService,
-} = require('../services/cartsService');
+const cartsService = require('../services/cartsService');
 
-const addItemToCartsController = async (req, res) => {
+const addItemToCarts = async (req, res) => {
   const { productId, sizeId } = req.body;
   const userId = req.decoded;
   try {
-    const ifAdded = await addItemToCartsService(userId, productId, sizeId);
+    const ifAdded = await cartsService.addItemToCarts(
```

Comment:
```suggestion
    const result = await cartsService.addItemToCarts(
```

서비스 메소드의 결과로 affectedRows를 반환하는데 ifAdded라는 이름은 어울리지 않는 것 같아요. 차라리 결과라는 표현이 더 알맞다고 생각됩니다.